cpp/include/raft/comms/test.hpp

```diff
@@ -230,12 +230,12 @@ bool test_pointToPoint_simple_send_recv(const handle_t &h, int numTrials) {
     for (int printrank = 0; printrank < communicator.get_size(); ++printrank) {
       if (communicator.get_rank() == printrank) {
         std::cout << "Rank " << communicator.get_rank() << " received: [";
-        for (int i = 0; i < received_data.size(); i++) {
+        for (int i = 0; i < static_cast<int>(received_data.size()); i++) {
```

Comment:
It is better to avoid explicit casting if possible and ```static_cast<int>(received_data.size())``` will break if received_data.size() > 2^31 - 1.

```suggestion
        for (size_t i = 0; i < received_data.size(); ++i) {
```